src/components/calcite-tabs/calcite-tabs.tsx

```diff
@@ -79,7 +71,8 @@ export class CalciteTabs {
   /**
    * @internal
    */
-  @Listen("calciteTabTitleUnregister") calciteTabTitleUnregister(e: CustomEvent): void {
+  @Listen("calciteTabTitleUnregister", { target: "document" })
```

Comment:
Should this be `body`? Or this could stay the same and have the custom event dispatch on the document rather than body.

Reply:
I'll use body for both